include/Core/Base/BaseModule.php

```diff
@@ -335,6 +335,9 @@ public function delete( $options ) {
 	 * @return string Success message.
 	 */
 	protected function get_success_message( $items_deleted ) {
+		if ( ! is_int( $items_deleted ) ) {
```

Comment:
Instead of checking the return value of `items_deleted`, let's set a flag at class level and use that for switching the message.

Reply:
Fixed.